openapi.yaml

```diff
@@ -15933,14 +15933,12 @@ components:
 
             Labels have the following constraints:
 
-              * Must start with an alpha character
-              * Must only consist of alphanumeric characters and dashes (`-`)
-              * Must not contain two dashes in a row
-          minLength: 3
+              * 4-byte UTF-8 characters are not supported.
+              * UTF-8 characters will be encoded in their byte representation. 
+              * If the label is using UTF-8, the response will return the value
```

Comment:
> The Unicode code point escape sequences returned by the API use up to 193 1 byte characters

Then give an example here of such a response (with less than 193 byte characters)